### PR TITLE
Web deploy fix

### PR DESCRIPTION
**Pull request by jburel**
Fix command and remove ref to empty file

### CODE REVIEW

**Comment by jburel** [comment]
@will-moore did you have a chance to look at it?

**Comment by will-moore** [comment]
Looks good. @pwalczysko You've been successfully testing these updated commands today, right?

**Comment by pwalczysko** [comment]
@will-moore : 

>You've been successfully testing these updated commands today, right?

So for example ``virtualenv omerowebvenv --system-site-packages`` is not included in https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.html and thus not run today by me at all.

there is ``virtualenv /home/omero/omerowebvenv --system-site-packages`` though which I ran - well, I am not sure this is a full substitute for testing

**Comment by jburel** [comment]
Looking at the page again, I will be more in favour of removing the commands since we now have the walkthrough links at the top and they do not offer much and add some confusion

**Comment by jburel** [comment]
@will-moore @pwalczysko I have removed the "default" configuration section
I don't think we need to keep it now that we have re-organized the page and added the links to the walkthroughs

File: omero/sysadmins/unix/install-web/web-deployment.rst [hunk 1]
```diff
@@ -232,9 +200,7 @@ a minimal configuration:
 
 
 and include this in your own manually created NGINX file, such as
-`/etc/nginx/conf.d/omero-web.conf`:
-
-.. literalinclude:: nginx-location-manual-wrapper.conf
+`/etc/nginx/conf.d/omero-web.conf`.
```
Comment (mtbc) [inline omero/sysadmins/unix/install-web/web-deployment.rst]: could be a later PR but maybe more of these should have `:file:` markup

**Comment by will-moore** [comment]
Makes sense. Looks good.

**Comment by jburel** [comment]
@mtbc done

**Comment by jburel** [comment]
Thanks merging

**Pull request merged**